app/src/main/java/com/uc/mymovies/service/UserService.kt

```diff
@@ -1,4 +1,5 @@
 package com.uc.mymovies.service
 
 class UserService {
+
```

Comment:
Is this class needed?

1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which is technical debt.